src/main/java/modtrekt/logic/commands/RemoveCommand.java

```diff
@@ -4,18 +4,24 @@
 
 import java.util.List;
 
+import com.beust.jcommander.Parameter;
+
 import modtrekt.commons.core.Messages;
 import modtrekt.commons.core.index.Index;
 import modtrekt.logic.commands.exceptions.CommandException;
 import modtrekt.logic.parser.CliSyntax;
+import modtrekt.logic.parser.converters.IndexConverter;
 import modtrekt.model.Model;
 import modtrekt.model.module.Module;
 
 /**
  * Deletes a module identified using it's displayed index from the module list.
  */
 public class RemoveCommand extends Command {
```

Comment:
Can we rename this class to `RemoveModuleCommand` for consistency?

Reply:
Nice catch. Renamed & also deleted the two `Parser` classes that are no longer used as a result of our port to JCommander.